app/jenkins-for-jira-ui/src/components/ConnectJenkins/ConnectJenkins/ConnectJenkins.tsx

```diff
@@ -36,7 +36,7 @@ const ConnectJenkins = () => {
 		try {
 			const { name, secret: retrievedSecret } = await getJenkinsServerWithSecret(uuid);
 			setServerName(name);
-			setSecret(retrievedSecret!);
+			setSecret(retrievedSecret);
```

Comment:
Not related to analytics but was giving a 'forbidden non-null assertion warning' in the terminal. Trying to clean things up bit by bit.